### ISSUE TITLE
Cursor moves to the front of "@" when using with Quilljs

### ISSUE THREAD

**Issue by ericbae** [zurb/tribute#86]
**How can we reproduce this bug?**

I have reproduced a very simple example in here

https://jsfiddle.net/03m5kyL7/

**What did you expect to happen?**

Mouse cursor should remain after "@" sign.

**What happened instead?**

Mouse cursor should moves to the front of "@" sign

**Link (jsfiddle/plunkr/codepen) or Screenshot:**

https://jsfiddle.net/03m5kyL7/

**Comment by ericbae** [comment]
I've made a progress. Previously, I was attaching the tribute object as follows:

`tribute.attach($("#component"));`

But, "#component" is actually not the contenteditable. When you create Quill object, it creates a contenteditable div _inside_ "#component". So, I've updated a [fiddle](https://jsfiddle.net/chphr0ab/1/) as follows:

`tribute.attach($("#component").find(".ql-editor"));`

So the problem of cursor jumping has now been resolved. But the behaviour is still a little bit weird. For example, pressing "enter" to select doesn't seem to work properly.
